paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -577,60 +578,6 @@ class PaymentSheet internal constructor(
         )
     }
 
-    @Parcelize
-    data class Address(
```

Comment:
This looks like a pretty large, intentional breaking change.

Reply:
reverted